evaluation/longbench/calculate_metrics.py

```diff
@@ -0,0 +1,221 @@
+import re
+import string
+
+import jieba
```

Comment:
Could you add a try-except block around
```
import jieba
from fuzzywuzzy import fuzz
```
that raises an error like "Please install jieba and fuzzywuzzy to run longbench"?

Also, some subsequent imports are unsued.